### PR TITLE
remove variable assignment from .env file

### PR DESCRIPTION
**Pull request by isabelizimm**
If users generate `.env` files with the specific variables CONNECT_API_KEY and CONNECT_SERVER, they do not have to pass them in to the `board_connect`, although they will still have to load the `.env` file.

### CODE REVIEW

File: class-work/01-intro-python.qmd [hunk 1]
```diff
@@ -89,15 +89,13 @@ board.pin_meta(___)
 
 - Create an API key on Connect: <https://docs.posit.co/connect/user/api-keys/>
 - Store it as `CONNECT_API_KEY` in `.env`
+- Store the url of your Connect server as `CONNECT_SERVER` in `.env`
```
Comment (juliasilge) [inline class-work/01-intro-python.qmd:92]: I believe Cole said this would be there automatically, right?

**Review by juliasilge — APPROVED** [review]
Thank you! 

When I changed this in the slides in 2e2767930d0d802eddea3a9a5f19fa54efc1dd16, I just took out all the dotenv code, which I tend to think will be better for the slides at the front of the room, so long as this is all here in the Quarto files. Let me know if you think I should add that back to the slides!

**Comment by isabelizimm** [comment]
> Let me know if you think I should add that back to the slides!

I would add it back, at least for the first time users see `board_connect`. It's only 2 lines now, rather than 5ish, and that way people following along know those lines are not in the class work by accident 😄

**Comment by juliasilge** [comment]
Makes sense! I added back and fixed up the other slides in e23fa1f98caf3224d9f9135cc6b48a3596cbfce5. 👍

**Comment by isabelizimm** [comment]
Oh no now I just added it back too!! Let me merge from main to make sure we don't have double load_dotenv

**Comment by juliasilge** [comment]
My push-to-main lifestyle in this repo has come to bite us! 😬

**Pull request merged**